keps/sig-node/20190131-runas-groupid.md

```diff
@@ -13,8 +13,8 @@ approvers:
   - "@derekwaynecarr"
 editor: TBD
 creation-date: 2017-06-21
-last-updated: 2019-02-14
-status: implementable
+last-updated: 2020-09-09
+status: implemented
```

Comment:
A little confused. this feature is going GA in 1.20? If so, then it's only marked as implemented once completely finished and then the underlying issue is closed.  Seems that this should stay implemetable.